src/doc/rustc/src/platform-support.md

```diff
@@ -305,6 +305,7 @@ target | std | host | notes
 `riscv64gc-unknown-freebsd` |   |   | RISC-V FreeBSD
 `riscv64gc-unknown-fuchsia` |   |   | RISC-V Fuchsia
 `riscv64gc-unknown-linux-musl` |   |   | RISC-V Linux (kernel 4.20, musl 1.2.0)
+`riscv64gc-unknown-netbsd`(platform-support/netbsd.md) | ✓ | ? | RISC-V NetBSD
```

Comment:
```suggestion
[`riscv64gc-unknown-netbsd`](platform-support/netbsd.md) | ✓ | ? | RISC-V NetBSD
```

Reply:
Thanks for the fix, I've applied and pushed it, ref. below.